.gitignore

```diff
@@ -9,3 +9,9 @@
 /tmp/
 Berksfile.lock
 .kitchen
+
+# Jetbrains IDEs
```

Comment:
As @jquick mentioned, you could add this to your global git config. Otherwise we face the challenge that we have to manage all editors available.

Reply:
Cool. Will remove!

Reply:
I agree, this needs to come out.

Reply:
Removed!